{{cookiecutter.repostory_name}}/noxfile.py

```diff
@@ -111,7 +117,8 @@ def format_(session):
 @nox.session(python=PYTHON_DEFAULT_VERSION)
 def lint(session):
     """Run linters in readonly mode."""
-    session.run('pip', 'install', '-e', '.[lint]')
+    # Check if lockfile is synchronized with pyproject.toml
```

Comment:
seems like this check is done always in install(...)
```suggestion
```

Reply:
Thanks. I wrote this comment when I was doing the check here instead of `install` function, forgot to delete.